### PR TITLE
[WIP] Create the coordinates module

### LINKED ISSUE #115: BL and Normal Coordinate Transformation : Confusion in Names!
🐞 **Problem**

There is lot of confusion in coordinates transformation! 
`bl_coord_transforms` and `coord_transforms` are not good names as in the `coord_transforms` does not specify the type of coordinates it works on while `bl_coord_transforms` clearly tells user what the module does! There is a need to make the coord transforms more intuitive.

🎯 **Goal**

Make coordinate transformations more intuitive in EinsteinPy

💡 **Possible solutions**

* Rename `coord_transforms.py`
* Merge both files into one
* Think of some other solution?

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #115`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### LINKED ISSUE #121: Clean the Messed up utils!
🐞 **Problem**

The `utils` module is currently too messed up. 

🎯 **Goal**

💡 **Possible solutions**

Make necessary changes to clean it. Refactor modules, code.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by shreyasbapat**
Fixes #115 

Partially fixes #121

### CODE REVIEW

File: src/einsteinpy/coordinates/core.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+import astropy.units as u
+
+class Cartesian:
+    """
+    Class for Cartesian Coordinates and related transformations.
+    """
+
+    @u.quantity_input(x=u.km, y=u.km, z=u.km)
```
Comment (ritzvik) [inline src/einsteinpy/coordinates/core.py]: Keep things in `u.m` by default. We follow SI units

File: src/einsteinpy/coordinates/core.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+        return Spherical(r, theta, phi)
+
+
+class Spherical:
+    """
+    Class for Spherical Coordinates and related transformations.
+    """
+
+    @u.quantity_input(r=u.km, theta=u.rad, phi=u.rad)
```
Comment (ritzvik) [inline src/einsteinpy/coordinates/core.py]: Here also, change it to `u.m`

File: src/einsteinpy/coordinates/core.py (hunk 1 above)
Comment (shreyasbapat) [reply src/einsteinpy/coordinates/core.py]: We are dealing with orbits, no-one would ever give parameters in SI units. km is a convenient unit.

**Comment by shreyasbapat** [comment]
Overridden by #176 
Closing

**Pull request closed without merging**